ci/jenkins/templates/trigger-upgrade-from-4-2-daily.yaml

```diff
@@ -0,0 +1,60 @@
+- job-template:
+    name: '{name}/{platform}/trigger-upgrade-from-4-2-daily'
+    project-type: pipeline
+    number-to-keep: 30
+    days-to-keep: 30
+    schedule: '3-5'
+    triggers:
+        - timed: 'H H({schedule}) * * *'
+    parameters:
+        - string:
+            name: BRANCH
+            default: '{branch}'
+            description: The branch to checkout
+        - string:
+            name: PLATFORM
+            default: '{platform}'
+            description: The platform to perform the tests on
+        - string:
+            name: DOWNSTREAM_JOB
+            description: Job to be triggered downstream
+            default: '{name}/{platform}/upgrade-from-4-2-daily'  
+        - string:
+            name: WORKER_TEMPLATE
+            description: Template for worker
+            default: 'maintenance-caasp-v4.2'
+        - string:
```

Comment:
is there any case in which we would need to add labels? As the worker is created just for the downstream job...seems like there is no need to add extra labels to it.

Reply:
Maybe not.